shopinvader/models/shopinvader_backend.py

```diff
@@ -88,6 +88,24 @@ class ShopinvaderBackend(models.Model):
     website_public_name = fields.Char(
         help="Public name of your backend/website."
     )
+    clear_cart_options = fields.Selection(
+        selection=[
+            ("delete", "Delete"),
+            ("clear", "Clear"),
+            ("new", "New"),
+            ("new_cancel", "New (cancel previous)"),
+        ],
+        required=True,
+        string="Clear cart",
+        default="clear",
+        help="Action to execute on the cart when the front want to clear the "
+        "current cart:\n"
+        "- Delete: delete the cart (and items);\n"
+        "- Clear: keep the cart but remove items;\n"
+        "- New: keep the cart (and items) and make a new one;"
```

Comment:
@acsonefho What is the aim of this ? How is it managed on session side ? The first cart is lost and won't be accessible on front side. Am I wrong ?

Reply:
In some website, we have a "clear cart" button.
But in Odoo, we have to tell which behavior we want.
For statistics reasons, maybe we want to keep cart not validated just to know what the customer try to buy (cases when we keep the cart; session cleared). Or maybe it doesn't matter (so we do a `unlink` (so session cleared) or delete of items (the cart_id from session is not updated)), etc.

Reply:
@acsonefho I agree with @rousseldenis What's the use of the "new" option ?

Reply:
@acsonefho IMO we should keep only 3 choices: Delete / Clear / Cancel. Moreover the help should explain each case on a functional point of view